app/assets/javascripts/deploys.js

```diff
@@ -98,7 +98,8 @@ $(function () {
     $ref_problem_list.empty();
     $.each(status_list, function(idx, status) {
       if (status.state != "success") {
-        var item = $ref_problem_list.append($("<li>"));
+        var item = $("<li>");
+        $ref_problem_list.append(item);
```

Comment:
previously only showed the last warning since .append returns the ul and then we put text into the ul ...